pkg/api/samplerserverv1alpha1/types.go

```diff
@@ -30,7 +30,11 @@ type MetricsNodeSampler struct {
 	PBPort int `json:"pbPort" yaml:"pbPort"`
 
 	// RestPort is the port to bind for the REST endpoint
-	RestPort                  int `json:"restPort" yaml:"restPort"`
+	RestPort int `json:"restPort" yaml:"restPort"`
+
+	// Address is used by tests to bind to localhost
+	Address string `json:"address" yaml:"address"`
```

Comment:
nit: Alternatively, add an `omitemtpy` so the tests above don't need to be modified

Reply:
done